source/nodejs/adaptivecards-controls/src/dropdown.ts

```diff
@@ -236,4 +268,12 @@ export class DropDown extends InputWithPopup<DropDownPopupControl, DropDownItem>
             this.selectedItem = this.items.get(index);
         }
     }
+
+    get parentLabelId() : string {
+        return this._parentLabelId;
+    }
+
+    set parentLabelId(newParentLabelId: string) {
```

Comment:
Nit: I tend to consistently use `value` as the name of parameter in a property setter.